docs/static/releasenotes.asciidoc

```diff
@@ -55,6 +56,43 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-13-0]]
+=== Logstash 8.13.0 Release Notes
+
+* Package `elastic_integration` plugin. https://github.com/elastic/logstash/pull/15769[#15769]
+* Flow metrics added `worker_utilization` to pipeline-level https://github.com/elastic/logstash/pull/15912[#15912]
+
+==== Plugins
+
+*Elastic_integration Filter - 0.1.6*
+
+* Using this filter you can process Elastic integrations powered by Elasticsearch Ingest Node in Logstash.
+** Once configured to point to an Elasticsearch cluster, this filter will detect which ingest pipeline (if any) should be executed for each event, using an explicitly-defined `pipeline_name` or auto-detecting the event’s data-stream and its default pipeline.
+** It will then load that pipeline’s definition from Elasticsearch and run that pipeline inside Logstash without transmitting the event to Elasticsearch. Events that are successfully handled by their ingest pipeline will have `[@metadata][target_ingest_pipeline]` set to `_none` so that any downstream Elasticsearch output in the Logstash pipeline will avoid running the event’s default pipeline again in Elasticsearch.
```

Comment:
```suggestion
* Using this filter you can process Elastic Integrations in Logstash.
** Once configured to point to an Elasticsearch cluster, this filter will detect which Elastic Integration processing (if any) should be executed for each event by auto-detecting the event’s data-stream.
** It will then perform that processing inside Logstash without transmitting the event to Elasticsearch. Events that are successfully handled will be tagged so that any downstream Elasticsearch output in the Logstash pipeline will avoid running the Elastic Integration again in Elasticsearch.
```

Notes:
 - zoomed up to high-level overview, skipping over details.
 - explicitly avoided mention of ingest pipelines, ingest node, etc.
 - explicitly avoided mention of `pipeline_name` setting, since this is a stop-gap feature for a specific subset of users that isn't useful in a general announcement.